api/rest_openapi.yaml

```diff
@@ -0,0 +1,335 @@
+openapi: 3.0.0
+
+tags:
+  - name: immutable
+  - name: mutable
+
+info:
+  title: Crasher REST API
+  description: Current API describes UI interaction with Crasher
```

Comment:
Do we consider "Crasher" as the whole system that consists of UI, language libraries and server? If so let's say with Crasher Server.